app/main.py

```diff
@@ -1,3 +1,135 @@
+import json
+import datetime
+from math import sqrt, inf
+
+
+class Car:
+    brand: str
+    fuel_consumption_one_km: float
+
+    def from_dict(self, data):
+        self.brand = data["brand"]
+        self.fuel_consumption_one_km = data["fuel_consumption"] / 100
+
+        return self
+
+
+class Product:
+    milk: int = 0
+    bread: int = 0
+    butter: int = 0
+
+    def from_dict(self, data):
```

Comment:
Usually, such method shouldn't set attributes but call a constructor creating a new instance of the class with the corresponding attributes